x-pack/test/security_solution_cypress/cypress/e2e/ai_assistant/attack_discovery.cy.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint @kbn/imports/no_unresolvable_imports: 0 */
+
+import { login } from '../../tasks/login';
+import { visitWithTimeRange } from '../../tasks/navigation';
+import { ATTACK_DISCOVERY_URL } from '../../urls/navigation';
+
+// TODO: Migrate to use Kibana Connectors API
+// @ts-expect-error
+import kibanaDevConfig from '../../../../../../config/kibana.dev.yml';
+
+describe(
+  'Attack discovery',
+  {
+    tags: [],
+    env: {
+      devConfig: true,
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'attackDiscoveryEnabled',
+          ])}`,
+        ],
+      },
+    },
+  },
+  async () => {
+    before(() => {
+      cy.task('loadAttackDiscoveryData');
+    });
+
+    beforeEach(() => {
+      login();
+      visitWithTimeRange(ATTACK_DISCOVERY_URL);
+    });
+
+    it.each(Object.keys(kibanaDevConfig['xpack.actions.preconfigured']))('%s', (connectorId) => {
```

Comment:
If there's no connector configured in the kibana.yml the test crashes in this line with:

``` > Cannot convert undefined or null to object```

Consider defaulting it (`?? []`) so cypress will tell there's no test to run instead, or maybe check this value and fail more gracefully if it's not defined. wdyt?